packages/react-components/src/components/HorizontalGallery.tsx

```diff
@@ -41,6 +41,10 @@ export interface HorizontalGalleryProps {
    * @defaultValue 5
    */
   childrenPerPage?: number;
+  /**
+   * helper function to choose which tiles to give video to.
+   */
+  setTilesToRender?: (indexes: number[]) => void;
```

Comment:
This is a callback, so a callback style name would be easier to understand, like onRenderingTile, or onDisplayTile (hmm, can't find a nice name, but it should be start with "on")